eng/build.sh

```diff
@@ -204,22 +209,42 @@ function BuildSolution {
   if [[ "$ci" != true ]]; then
     quiet_restore=true
   fi
+  fslexyacc_target_framework=netcoreapp2.0
+
+  # Node reuse fails because multiple different versions of FSharp.Build.dll get loaded into MSBuild nodes
+  node_reuse=false
```

Comment:
Same comment, where is this consumed?

Reply:
Yeah, as far as I can see `node_reuse` is accepted as a gloal setting by `tools.sh` `MSBuild`.  But my shell fu is not that great.  Could you check how we should pass this value through, and if it's appropriate to just set the global, or should we pass it through the `MSBuild` function?